app/providers.tsx

```diff
@@ -8,14 +10,18 @@ const TurnkeyIframeElementId = "turnkey-iframe-element-id";
 
 export const Providers = (props: PropsWithChildren) => {
   const [queryClient] = useState(() => new QueryClient());
-  const [clientConfig] = useState({
+  const [clientConfig] = useState<
+    Exclude<AlchemySignerParams["client"], AlchemySignerClient>
+  >({
     connection: {
       rpcUrl: "/api/rpc",
     },
     iframeConfig: {
       iframeContainerId: TurnkeyIframeContainerId,
       iframeElementId: TurnkeyIframeElementId,
     },
+    rootOrgId:
+      typeof window !== "undefined" ? env.NEXT_PUBLIC_ROOT_ORG_ID : undefined,
```

Comment:
learn: why is this needed again? for mobile?

Reply:
this is only relevant in an SSR context and not used on the server

Reply:
oh and the rootOrgId is used for us to be able to test against staging mainly. It's not really meant to be used by other entities and probably won't even work for them